Snakefile

```diff
@@ -14,40 +18,31 @@ rule files:
 files = rules.files.params
 
 rule download:
-    message: "Downloading sequences from fauna"
+    message: "Downloading sequences and metadata from data.nextstrain.org"
     output:
-        sequences = "data/zika.fasta"
+        sequences = "data/sequences.fasta.xz",
+        metadata = "data/metadata.tsv.gz"
     params:
-        fasta_fields = "strain virus accession collection_date region country division location source locus authors url title journal puburl"
+        sequences_url = "https://data.nextstrain.org/files/zika/sequences.fasta.xz",
+        metadata_url = "https://data.nextstrain.org/files/zika/metadata.tsv.gz"
     shell:
         """
-        python3 ../fauna/vdb/download.py \
-            --database vdb \
-            --virus zika \
-            --fasta_fields {params.fasta_fields} \
-            --resolve_method choose_genbank \
-            --path $(dirname {output.sequences}) \
-            --fstem $(basename {output.sequences} .fasta)
+        curl -fsSL --compressed {params.sequences_url:q} --output {output.sequences}
+        curl -fsSL --compressed {params.metadata_url:q} --output {output.metadata}
         """
 
-rule parse:
-    message: "Parsing fasta into sequences and metadata"
+rule decompress:
+    message: "Decompressing sequences and metadata"
     input:
-        sequences = rules.download.output.sequences
+        sequences = "data/sequences.fasta.xz",
+        metadata = "data/metadata.tsv.gz"
     output:
-        sequences = "results/sequences.fasta",
-        metadata = "results/metadata.tsv"
-    params:
-        fasta_fields = "strain virus accession date region country division city db segment authors url title journal paper_url",
-        prettify_fields = "region country division city"
+        sequences = "data/sequences.fasta",
+        metadata = "data/metadata.tsv"
     shell:
         """
-        augur parse \
-            --sequences {input.sequences} \
-            --output-sequences {output.sequences} \
-            --output-metadata {output.metadata} \
-            --fields {params.fasta_fields} \
-            --prettify-fields {params.prettify_fields}
+        gzip --decompress --keep {input.metadata}
+        xz --decompress --keep {input.sequences}
```

Comment:
Was there a reason you chose to a) decompress separately and b) keep the compressed copies around? My instinct would be decompress on the fly during download and thus make this whole `decompress` rule unnecessary and avoid the double disk space usage (which while insignificant for Zika, sets what I think is a bad precedent).

Reply:
I vote to drop `--keep` in order to remove unnessary intermediate files. Although I also vote for `xz` over `gzip` for a smaller memory footprint. ;) I have no strong opinions on "decompress on the fly during download", will follow the group decision. @trvrb feel free to comment on your flag decisions